.spelling

```diff
@@ -42,6 +42,8 @@ Budinsky
 CAP_NET_ADMIN
 CAs
 CDNs
+CDR
```

Comment:
CDR  and CDRs don't appear to be used in this document.  Why adding them to .spelling?  Did you mean CRD and CRDs?